src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -227,6 +233,21 @@ paths:
           description: A Metadata object describing this service.
           schema:
               $ref: '#/definitions/Metadata'
+              
+  /tool-types:
+    get:
+      summary: List all tool types
+      description: >
+        This endpoint returns all tool-types available
+      tags:
+        - GA4GH
+      responses:
+        '200':
+          description: An array of methods that match the filter.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/ToolType'
 
 definitions:
```

Comment:
FYI @briandoconnor 
reviewing the tool-registry-schema specification, I think we left the tool-type field in for future expansion (i.e. workflows, services)

In that vein, a workflow could be considered a degenerate tool with no associated image and dockerfile.